src/goto-symex/goto_symex.h

```diff
@@ -164,6 +164,17 @@ class goto_symext
   /// symbolic execution from that state.
   bool should_pause_symex;
 
+  /// If this flag is set to true then assertions will be temporarily ignored
+  /// by the symbolic executions.
+  bool ignore_assertions = false;
+
+  virtual bool check_break(
```

Comment:
> Documentation?

https://github.com/diffblue/cbmc/pull/4361#discussion_r292644774

outstanding

Reply:
Documented (and refactored a bit).